src/backend/Shared/Timebox.Shared.Requests/Interfaces/IRequest.cs

```diff
@@ -0,0 +1,8 @@
+namespace Timebox.Modules.Requests.Interfaces
+{
+    //Marker Interface
+    public interface IRequest
+    {
+        
```

Comment:
Some metadata here might be nice; particularly during development. Things such as the call number, where its from etc.

Reply:
The marker is there just to state in generic constraints we could store these values but they'd be logged and collated by another module perhaps and stored in a cache or database